### DISCUSSION TITLE
Can we use fsearch in terminal ?

### DISCUSSION THREAD

**Discussion by q2333gh** [cboxdoerfer/fsearch#539] [General]
I really like Everything in windows. And im gradually also familiaring with linux terminal. Great productive stuffs.

So can we achieve similar goal like using Everything in Linux terminal with almost the same instant speed ? 
Thats so cool!

**Comment by q2333gh** [comment]
Also i can try to read some of fsearch code hah.

**Comment by RugnarLodbrok** [comment]
there is https://github.com/junegunn/fzf, as far as I see, it scans entire fs on each startup. And unlike `everything` it can't monitor fs changes in real time

**Comment by zacaj** [comment]
This would be really handy, feel like I've got 2-3 different file databases sitting around at this point between FSearch, Dolphin/KDE, command line, etc

**Reply by q2333gh** [reply]
> This would be really handy, feel like I've got 2-3 different file databases sitting around at this point between FSearch, Dolphin/KDE, command line, etc

yeah. with the help to latest SOTA code agent. we might fork fsearch to make a cli version of it !
